astrocut/tests/test_cutout_processing.py

```diff
@@ -247,14 +247,18 @@ def test_center_on_path(tmpdir):
     assert primary_header["OBJECT"] == "Test Target"
 
     # Using the default output filename and not giving an image wcs
-    out_file = cutout_processing.center_on_path(path, size, [cutout_file],
-                                                output_path=tmpdir, verbose=False)
+    target = "C/ Targetname" 
+    out_file = cutout_processing.center_on_path(path, size, [cutout_file], 
+                                                target=target, output_path=tmpdir, 
+                                                verbose=False)
     assert "path" in out_file
```

Comment:
This assertion I believe was attempting to test this line of code when target is unset: 

https://github.com/spacetelescope/astrocut/blob/c0d3ab50d0c3a2ddab07fd8ac0fd3619164e9852/astrocut/cutout_processing.py#L425

However, you're replaced the test there (about using the default output filename, and the reason this still passes is because of a side effect of the termporary directory that pytest creates -- it contains the word "path" (see `tempdir` fixture in this test).

For me, `tmpdir` is: `local('[/private/var/folders/d5/7lllrg_s1238gsx2j2ng9bqw0002h3/T/pytest-of-bfalk/pytest-0/test_center_on_path0]()')`

path is included in the name of the directory, because pytest includes the name of the test, which happens to include the string "path".

I think some of this might be clearer and easier to resolve if these different calls into `center_on_path` were parametrized as I was suggesting in a different comment.